conpot/protocols/bacnet/bacnet.xml

```diff
@@ -0,0 +1,39 @@
+<bacnet enabled="True" host="0.0.0.0" port="47808">
```

Comment:
I guess this file should reside in conpot/templates/default/bacnet/ rather than /conpot/protocols/bacnet :)